examples/hello-world/step-by-step/readme.md

```diff
@@ -0,0 +1,52 @@
+#  Step-by-Step Examples
+
+When give an machine learning problem, we probably wonder, 
+where do we start to formulate the federated learning problem. 
```

Comment:
When given a machine learning problem, we probably wonder, 
Where do we start to formulate the federated learning problem?